create_rental.rb

```diff
@@ -20,6 +20,7 @@ def create_rental
 
   rental = Rental.new(date, @books[selected_id_book], @people[selected_id_person])
   @rentals.push(rental)
+  save_rental(date, @books[selected_id_book], @people[selected_id_person])
```

Comment:
- [ ] you are saving to the json file each time the rental gets created, the issue with this is that after adding the rental and then showing the list of rentals the added rental will appear twice. you can solve this by updating all the json files when the user is closing the app.